local/.utils

```diff
@@ -23,12 +23,12 @@ wait_command() {
 
 cleanup() {
     if [[ -n "${KCP_PIDS}" ]]; then
-      printf "\nCleaning up processes $KCP_PIDS\n"
-      kill $KCP_PIDS
+      printf "\nCleaning up processes %s\n" "$KCP_PIDS"
```

Comment:
These updates to `printf` are based on [recommendation by shellcheck](https://github.com/koalaman/shellcheck/wiki/SC2059)